### DISCUSSION TITLE
Animals trough portals?

### DISCUSSION THREAD

**Discussion by zeuzi** [Zrips/CMI#1681] [Feature Requests]
**Description of issue or feature request:**

Please add the posibilty to go trough portals with animals (all enteties)
maybe add a config for not allowed animals to go trough and a possibilty to make blacklist as whitelist.

1.13.2

**Server Version (using `/ver`): 1.13.2-R0.1-snapshot Craftbukkit

**Comment by Blueeyestar** [comment]
Taking animals through the portals, especially if you work with Multiworld, would really make sense. Many users always ask if you can take animals with you from the world of farming..unfortunately this is not possible.

**Comment by zeuzi** [comment]
> Taking animals through the portals, especially if you work with Multiworld, would really make sense. Many users always ask if you can take animals with you from the world of farming..unfortunately this is not possible.



This cant be true since youre able to teleport with youre animal when doing so on a command
for example:  You are riding a horse and doing /home it'll teleport you with youre horse,
so why not just use portals with commands so when going trough a portal it executes the command /tppos (coords to other end of portal)

**Comment by Zrips** [comment]
What portals are we talking about here? CMI portals or vanilla ones? And in general, some different approach would be needed, as tracking each and every entity when it enters portal area can be quite heavy on server side. Maybe simple "packing" animal into spawn egg would be better approach.
